service_container.rst

```diff
@@ -311,18 +313,19 @@ made. To do that, you create a new class::
         {
             $happyMessage = $this->messageGenerator->getHappyMessage();
 
-            $message = (new \Swift_Message('Site update just happened!'))
-                ->setFrom('admin@example.com')
-                ->setTo('manager@example.com')
-                ->addPart(
-                    'Someone just updated the site. We told them: '.$happyMessage
-                );
+            $email = (new Email())
+                ->from('admin@example.com')
+                ->to('manager@example.com')
+                ->subject('Site update just happened!')
+                ->text('Someone just updated the site. We told them: '.$happyMessage);
 
-            return $this->mailer->send($message) > 0;
+            $this->mailer->send($email);
+
+            // ...
         }
     }
 
-This needs the ``MessageGenerator`` *and* the ``Swift_Mailer`` service. That's no
+This needs the ``MessageGenerator`` *and* the ``Mailer`` service. That's no
```

Comment:
`MailerInterface`

Reply:
Idk, I'm a little sceptical about this change. We call the `Mailer` service through the `MailerInterface` no?

Would't this sound better ?
> This needs the ``MessageGenerator`` *and* the ``Mailer`` service. That's no
problem, we ask them by type-hinting their class and interface names!

Reply:
Sounds better indeed